lib/led/rgb.js

```diff
@@ -429,19 +479,22 @@ class RGB {
    */
 
   [Animation.render](frames) {
-    return this.color(frames[0]);
+    const state = priv.get(this);
+    state.value = frames[0];
+    return this.update();
```

Comment:
The `Animation.render` method here was calling `color()`, which does a lot of normalization and validation before itself calling `update` on line 270 above. 
Now the `render` method behaves more like `render` in the [`led class`](https://github.com/rwaldron/johnny-five/blob/main/lib/led/led.js#L334) which 'trusts' its animations and calls `update` directly .
This also bypasses the reassignment of `state.prev` to a 'tween' value when `Animation.render` is called for `pulse` function which resulted in the RGB values decreasing so that the RGB light would gradually decrease.